### ISSUE TITLE
[Experiment] Hyper-parameter tuning

### ISSUE THREAD

**Issue by km9mn** [boostcampaitech3/level2-object-detection-level2-cv-09#55]
### What?
- SwinL Cascade hyper-parameter tuning

### Why?
- 성능 향상

### How?
- cross validation fold 1으로 train, val 후 wandb에서 성능 비교

### Todo
- [ ] batch size (samples_per_gpu)
- [ ] epoch
- [ ] optimizer
- [ ] learning rate
- [ ] learning rate scheduler
- [ ] loss
